src/app/api/dashboards.ts

```diff
@@ -1,38 +1,25 @@
 import { getCookie } from '@/utils/cookies';
-import { DashboardEditResponse, ErrorMessage } from '@planit-types';
+import {
+  DashboardEditResponse,
+  DashboardFormData,
+  DashboardListResponse,
+  DashboardResponse,
+  DashboardUpdateData,
+  ErrorMessage,
+} from '@planit-types';
 
 export const API_URL = process.env.NEXT_PUBLIC_BASE_URL;
 
-type DashboardResponse = {
-  cursorId: number;
-  totalCount: number;
-  dashboards: Array<{
-    id: number;
-    title: string;
-    color: string;
-    createdAt: string;
-    updatedAt: string;
-    createdByMe: boolean;
-    userId: number;
-  }>;
-};
-
-type DashboardUpdateData = {
-  title: string;
-  color: string;
-};
-
-type DashboardFormData = {
-  title: string;
-  color: string;
-};
-
 // 대시보드 목록 조회 - GET
-export async function getDashboards(): Promise<DashboardResponse> {
+export async function getDashboards(
+  navigationMethod: string,
+  page: number,
+  size: number,
+): Promise<DashboardListResponse> {
   try {
     const token = getCookie('accessToken');
     const response = await fetch(
-      `${API_URL}/dashboards?navigationMethod=infiniteScroll`,
+      `${API_URL}/dashboards?navigationMethod=${navigationMethod}&page=${page}&size=${size}`,
```

Comment:
이부분은 
```suggestion
const params = new URLSearchParams({
    navigationMethod: navigationMethod.toString(),
    size: size.toString(),
    page: cardId.toString(),
  });
  
 const url = `${API_URL}/comments?${params}`;
```
이렇게 해도 좋을 것 같아요!